plugins/goodreads.koplugin/doublekeyvaluepage.lua

```diff
@@ -176,7 +176,7 @@ local DoubleKeyValuePage = InputContainer:new{
     show_page = 1,
     text_input = "",
     pages = 1,
-    goodreadersKey = "",
```

Comment:
Oops, I missed this part. I know you did not write this line, but could you rewrite this variable name to `goodreads_key`?